pkg/server/server.go

```diff
@@ -200,6 +202,33 @@ func Run(logger log.Logger) error {
 		func(error) { serverInsecure.Shutdown(context.Background()) },
 	)
 
+	{
+		// We clean out old instances that have invalid tokens periodically as access
+		// to the PostgreSQL instances only relies on certificate authentication. This
+		// means that is situations, such as a user being offboarded, they will lose
+		// access to the draupnir, but not their instances.
+		logger.With("component", "cleaner")
+
+		service, err := google.New(http.DefaultClient)
```

Comment:
I actually meant this could be constructed in the `NewInstanceCleaner` function, but also, it can be here too without any issue. Just depends on where you want to locate the context for how you construct the service!

Reply:
Ah right. I think if eventually writes tests for this we might refactor it. For now leaving as is.